wp-content/themes/oceanwp-child/single-blog_post.php

```diff
@@ -17,32 +17,32 @@
 
 	<style>
 		.single .entry-title{
-    display:none;
-}
-.wpulike-heart .wp_ulike_general_class{
-    background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
-    box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
-    border-radius: 30px !important;
-}
-
-.wpulike-heart .count-box{
-    color: #fff !important
-}
-
-.ess-all-networks-button, #ess-main-wrapper .ess-total-share{
-    background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
-    box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
-    border-radius: 30px !important;
-}
-
-@media only screen and (min-width: 960px){
-    .content-area, .content-left-sidebar .content-area{
-        width:75%
-    }
-    .widget-area.sidebar-primary {
-        width: 25%;
-    }
-}
+			display:none;
+		}
+		.wpulike-heart .wp_ulike_general_class{
+			background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
+			box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
+			border-radius: 30px !important;
+		}
+
+		.wpulike-heart .count-box{
+			color: #fff !important
+		}
+
+		.ess-all-networks-button, #ess-main-wrapper .ess-total-share{
+			background: linear-gradient(80.63deg, #0B4F6D 26.04%, #0F7AA9 99.31%) !important;
+			box-shadow: 2px 3px 9px rgba(0, 0, 0, 0.2) !important;
+			border-radius: 30px !important;
+		}
+
+		@media only screen and (min-width: 960px){
+			.content-area, .content-left-sidebar .content-area{
+				width:75%
+			}
+			.widget-area.sidebar-primary {
+				width: 25%;
+			}
+		}
```

Comment:
Is this true for all pages? Or just the blog?

Reply:
Just blog that's why I've put it there.

Reply:
Isn't it true also for initiatives page?

Reply:
No, I'm using bootstrap grid there.